examples/chainer_simple.py

```diff
@@ -108,7 +111,11 @@ def objective(trial):
 
 if __name__ == '__main__':
     import optuna
-    study = optuna.create_study(direction='maximize')
+    from optuna.integration import ChainerPruningExtension
+    # This verbosity change is just to simplify the notebook output.
```

Comment:
[nit-picking]
In my feelings, the word "script" is more natural than "notebook" because this example is not always executed on Jupyter Notebook or  Google Colaboratory:
```suggestion
    # This verbosity change is just to simplify the script output.
```